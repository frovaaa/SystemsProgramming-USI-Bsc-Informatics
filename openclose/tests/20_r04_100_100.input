AaAaZOoEXDPpVN2RWwrFS2QTJjGOTt4WVKkDBHJjhbdVAa4IEeORJjPPppRBOoKVv0OoUQ6qukbrroivvwogtqsfnvdxez
AaWHQBOoCHXxEeWwhcSsMmb6XDQXFUGQWKFU0LlUOWLJjYHECOoFABDIGgOGgZBbPYypzoidbafcehylwouufkwqgufxqdxqhw
Gg6MJCRrcOUuTtZHPPTMNn4ZL5QAaqFfWJ3ZzIXIW38P9KI3WwDdiPT0tO49LSRrMmslopkpwixijwlzmtpphzojm
LGG7gER2PpYTtMMQqUZ3zuLlmUTWwt88ANS5GRXD0GQYWwDFCZzYBDROWYHCWSswchywordbycfdyqgdxrgsnaumyregl
KORrNCMmcnXxHDdVvhHW3SswhPIipAJFLlfWSswjXxa4HhO0IV1ANnaZLlMSKkAIDWIZziwdJFEBZXTtxzbefjiasmzviook
WwWLZzBF85HT87thRr7LVJjvTWwXWUWJYNnLRTPTt13BHD06QUDCWGFEefEP25pegwcduqdhbptrlyjwuwxtlfblw
JHhjVvKkOBbKk6HhKkQ9qIYyioHP8Hhp3GP0EeJCOQZQMmqLDGgESsIAaJOCcHLXxlXDdxUJRTIitrjuhojiedlzqocjpgh
NEM3Z3zZzWw2mFVTOGXIDdiKkOJBRr6XxbTBQqWRrPLl2pPpwbASBWAaw4SKkWPPFfZzpVvpwYXKkxysbsatjoxgotvfen
QYyRVGLLOAQqXxCcXIKG8QOAUuSKksVLlWZIUXG3HXGgGLlFfgC6TUKPSspkUCBXKkxbcuutcxhgxuizwvaoqgkixaollgvrq
2YOTtZCcZC60WwWwKUQODJITNAaZNP1GgCOUDHPphLlSNFSsCCGgc3XXH0KkhLGRrglxxcfnsduocpnzntijdoqukczzoy
LlSQBbW5wBDXZFWwYZzQqyfHXxhQXxTtJjJj8PpIiHhqSsOTto1PpW7wPVPpvUupJRWO9owHBLLQq8EeWwllbhrjzxdbqs
PpSIiZYyJJ2YFWFAaHZESsDIZzUY3MGgmXxyuNRFAWHh9CVPLBCPVJjGgv6C5QXxBTtbqcpcblpvcwafrnidezhfwfyjjzs
HIYyHhHYNnMmy1WAQqQqFCcfYyRraWINDYYyydCXMSTTYytt2DSsDdLlXxEFYyEWwHBFUuSTJj0tsLlfbhefedsmxcniwwhih
TJOHhoSsTtjMmtMAaVvNXxTCOTY6SVvsTTtXT5Cc6GUuVvgBRrbUuNIiGUOTPG7INS1sYFERrefynigptougntxtytoctnm
PVGJjGLlgOoJjgIFf6UVHh1IiHTJAGYMOQOoqoKBZzD3dE3F1TOKVZzQqCVVvvXX1JT0tjxxcvkotfebkmygajthvuivp
ZUu4DdHOVvoANVWOoICcNMmT1GOLEYXZzNLlBbnSNnTTtNGPNnZOXOoPJjAaDIABRrb07aidpxozpgntsxyelogtniwvnahz
IVJST9CMUOEFWQM2YyMLEADGgRQCcWwGgqJjWVIivwWwQDES1FVHh8vGDdgfsUGEBITtibeguedqrdaelmmqwfeoumctsjvi
Tt6EePA378HhPGITSOGgG7VKIZzikvgKH1hkosRrF8FfR7CR9JjVNnD5dvBbJSZFYPBbpDd7Qqyfzsjrcrftigpap
KNEUueZznKZGGHh6gCDMmXKHPphDdIMX2EVve8QqLZ3QQ7EeCUGgEJjNnABbHAN8ZzQJNnjqnahaeucqqzlxmikxdcgzkk
ZPRTtABQ3GLlSCIBJjHhBSsbbTtITOoAaLC5Jj3DUAa4LXGITFWUEIiIXXxXOoXxUOouxxieuwftigxludcltiicsgqbarpz
Kk4CVPGgMAWXx1FGCSsVvDdIiQJjSSF6WFWwRYyL6S8PKkGD9AYDCcBbYWVSs9IiYyvwydyadgpslrfwfssqcgfwampvc
HhWNFXFIi1CcHPphMmOEeoC5cZ6zZzNCOHEYyJj6O7ZGgKkSYKWwRrQPMmLPOQPEFFROorffepqoplpqkyszoehocnfxfnw
KTtFVNnvVUuOoSYRTAVvat5VOEVKHAEeaHR85WWGgANnHhaXxLJjPWRLl8RQqrJjQqrXWGgH3hwxwplwwrhhkveovrysvfk
RIiSsXGXOXxUuVVAavvCAacTLZzl5QqCPpIiZz4CAEUDBTBb0THhSsJjPptWwCJXxjZGSLCTMCcmtclsgzctbdueacctoxgxr
YIiKQqkySNZPDTtWWPWwMmLEDVSXUuXxxsZVRrBOobOLlovOLPJjDPPTtpDCcFfdVYyLWHOoTUCcuthwlvpdplozvdelpwwdpzns
RrUJMFNnHhKTtAaTYZzyPQTE5eBNAQqW8PFfUuDFTAatYZTEGNAMmOoaV2SR3XOD93doxrsvngetzyfdpwanbtqptkfmju
3ZDZEMTtmJZYQLWwXxWZzQqwYyIiHFDGgdfNXxnhRrlqYFMXKC65TtYGFOQUuGgVAYGGggACZzcayavqofgyckxmfyyzjezdz
WNSJRUNBFfWCcQVUuEJjTteLEeBCHhMmLTtHhlLEHhEBZ8JjG5NFfDdJjZKUHh1u8BbDdkzFLlfngzbeelcblvqwbnurjsnw
GFfgWWVFIUEeuLETUZzUBJFCMmMXxmIiCccIMmIOoFYYyR7JAWwaQPJKkAajpqPpLIWCcYUUVvuuywiljryfiifjbuutelifvww
R1YYPBEFfXxeb30VRIiUY9YEeCMCMACGTGgZDdDAKkadOPW266NJjQqnWCcNHMCUuJjcmhnwwpoztgcamcmcyyurvpyyr
GgPTtZKkMmzMQFS26L2DUuIidNJjKWAVPGgpLOolSXxO42HDOoAUuaNTOotTtICIYyL1CJVvjclicindhosvawknlsfqmp
I4V8JjWw8SsKkPMSPBbBM7mCcWwbpXIixPTTBbtDYydtp2sLKKGDdLlgkkFfNnB5HhblED7VSsYBLQqORrolbyvdempvi
ZBAabWK0ZzGYTNntL8ZFWCcwJYU7AHhAXQAZzSOoIHARJXVPpOYJjyPpALCcWZzAawGWwglaovxjrahisaqxaauyjfzlygkwz
PpVNDdnQqXxvXQEJjeFQLlqfN7IPDdpRrB7AaZzXxQqb4in8QqRrNHRFCXCRDIABbZZJUO1EeoJEejujzzaidrcxcfrhnqx
JjBNREP8RP9DdPDDOSYLAa5NOKMPA8UTtBbFfDWJUNnJj8uBHADHhNPpnEeEQUuqQqedahbjwduapmkonlysoddpprpernb
SPKDdVvTIitVvkpIVvKTtkPLFfJWwjWwlJZzVA0TLLDKkdlltB0bFD3KkdfaYD6d72Ppy8UFFNXEOOMmooexnffuvjpis
PpSEQPD9HD8ABWwbMWwDdZMmQPpqzQOPpAAaNnHFIifhPWTtGgYLlQ16BbEeOYyGg3BVWFTAatfwvboqywpaoqmadhdpqes
4VPpDd94QWwTPESD0d4C1JBbRWw0KkGTM6WwUJYyFRBWGgEeSJREeRBQTt9MT9Jjtmqbrrjswbrfjumtgrjcseptqv
PF8HPDdWSQKkFfFfTKC2OLRCQqJj3FFfHWwAVv5PAapIUuiLVvNHPpXxZFOBHhHVCcIOoivhbofzhnlahfcrlocktqswphfp
OoAKSOKBZJHhTQqQK5BAYy6YFJXNnKZzS5QKZKkYLl2EQqeR4KkBbryWELLKKkNK1knkllewzkqskxjfyabkqtjzbkoska
OoAaEe7XxFfGNW6PpKkZSszLUQLIilqulHhwJjnHTQqONnoXxVvTtHhFfI743DGgZAaSszdiLPROorplWCcwYXxyRVvrthg
Ee1NnSCcsPYypXxKkOLQVvqNUunloPZzp1EeIiXNGgnxK2RN4Y28EQFLRPTBTQUIONRrnMZSKkszmoiuqtbtprlfqeynrk
RJTZOPAaFPpfFCLlcSIQYy2PGSSSUuOTDdEG8QFVQTt1VFWLROLUuLVREeIORroirvllorlwfvqvfqgetosssgpqisfpoztjr
JUuNnHhXIiTIiLltFIZOLloKACCUuIicK6kcHhYI3ABLLHhlBGg5DCZzcVvdNnKCWwcCckbMmHF1XTNntxfhlbaiyakzifxj
AYKOokMRYOHYyXXGgDPPFfYyP2OFXMG3OoJRrCIZAaEKWwkQC1Bb9TIiASPNnXxpIisaGgtcqezicjgmxfopppdxxhoyrmya
HCAGSsTtFIiQqfJCHLHIi4AHCcYyR4DdBMTVGR4YQqGgyrgvtmHYWOUuTtoQDPOKZCczHJYyMmjhkopdqwyhbrhahlhcjgach
UMZzVZGJSVvRLlBAaOTHhE7CBZJjLlzXUuCcxXSS8HUuYIUuBPpbDBOJjQqAQEUEOoeuEXxeeqaobdiyhssxbcetobrsjgzvmu
1QITPXxFAYDKkBT9FAFUWw0UWwuGCcDSMms3dILJjl9ZzCcEeUSsCcSJ5KkjFfDFNnf6GOQqogdsuigufaftbdyafptiq
8YJIKkRYOoIGgQBIVBFfbMDSsGgBb5EEKYyTB0Jj5btPZOXUHATNSsOLGVYyVFPKkpfvvglontahuxozpkeedmvibqiyrijy
PpLMCZzOPRrC0CQqGIiCOoXOoxO7TKMPUuYyRrpHVvIMTOSHUuhsotPp1DSBbKFfk5ZAazSTXEextssdmihmktocgccpocml
JBRKkRWw1RrAJAB4bBbEeCcRrHhRPMWwmRQqOAaMMH8BbFCFfUuFHHXxhhfJIijcZVOHMK5LVvlkmhovzfhmmorprajarrbj
A9VVHZzCUL0QZzSsLUMmKTRrBZZzzBbSRrKPLETtP4CcGgpVvKkeLUHPRTtQqrZF3KDdADdakfzphullpksbtkulqluchvva
MmGg1PXxpHJNGOognjXTNDdGgFfGCKODFQIPOCcTBOoIARQMMRrMA4VEeYZSsCDVHDdhvdczyvammmqraibtopiqfdokcgntxh
RrWRrwTBRATBMYJjAaWAMLCcTK2WwCc1CcDQqdXxJjkXxCcX8QqAT6V7IiQIJjNSsHNBJVvjbnhniqvtaxtlmawymbtarbt
M311VvQTRrI7TYH9CchPpytNUWwHWIiKkXxPBbDSOUOoYYyyVAGYBbXxCcZ2DdzY3PFNnZzAafpyygavuosdpwhunitqm
ZCczEZKFWwICKkYycPMmpGL2RTMD1GgWHhCOIiocYYWDMmPpdYQDdqENBbHVv2HFO2ofhHHhRBbrhhneywyywdmtrlgifkze
YyUuADPpMmILZTRrSKkXRDdrxAKDECWKkwLYylADCJjcdaKkSCMXxIiCHCcVXxVBLIW4FfDLVvldwilbvvhcmcscedkastzlida
WA6JjOo3DJjPpEQDdSUIYyiPp6PBb8KHhkpHhTtJMIi8TNCFfcnAGRHHLHPpFMCVNSNO8onsnvcmfhlhhrgatmjusqedaw
UV7PXIWFJY3ZAFOofRHhraJjILNnOUBXGC5OEeoIIXDdGAaUugYCGQqBbKCCc02S3Ggsckgcyxiicgxbuolizyjfwixpvu
FVvGgY5VvQNHJ0FZzJMUumDSOo6NA0FZWKDdkNCJXxHhjcJVCM3TKLl9JjDdMmGX08Bbxgktmcvjnwzfansdjfjhnqyf
Qq5BNnKLlkbJDMZzGgmdLTtNVFUufvHhnUVJHPBbphZVF0VHhDdBYybvfQqvzIBbXx8XF0557ZZQJQGgqjqzzfxijvulj
TMNnmD7dtPOLSBbCOocAasEIHhZIAMOoMRrmmaUuL4H59FfDdBUuTtbHhhJQXNMGgmCJSsBbATTtGTtgtajcnxqjlizielop
LHhQqlQ0YyRTtBAaEIWPpM2DdmGgHOoRRrXF2KFDSQZDZXIHYJGgEIiCcejyXxhZMmDZRQGgqrzdzixzdzqsdfkfxrhwiebrq
0LXAaPQNnqMNn3EeC21JjQqE0DMPBbKCck9SAUuKEJjSMEem5VMQWwqB2A48ZC9ZQZzqzczabmvsekaspmdecmpxl
QE5LRrORrTGT73HhtgHUuZBUVvuQGgqQOoQLUulSSFXGNU3ZzPpQ3SWYyFNBUQTYHhYyytqubnfwsqungxfssqqbzhtoleq
ZJ0YyVMYZAaUuJZFKkOo3SsTKD4EWweOITABK3DdIik0baIitiKV7HNnhUSS4HhHQETt4Nneqhssuvkodktfzjzymvjz
VQWwEeWCCcNnJjNMLAalNSCLNF2fBUZ6zu9VXKSZOoMmN8FRTDdFTOJBMCYyXICYycixcmbjotftrfnzskxvbnlcsnmncwqv
Ee1A3VvaJjTQKTWIQKKW7IJN5VvRAWwarOXxonj2NB8Xx25HRUHhuTtAaCMmcr7hV4UuLDA0adlvbniwkkqiwtkqt
RQVNCAaMVXxvCNXxncAVAV6YyUOZYyCcIWwizoCVVMmAaUJFBbVvOofjuRrv8XZzW3VXDPDMmNPpndpdxvwxvcuvavamcnvqr
VKkFfDWw4PKTJjt8kEVBCPpcBODRrdo60FfQLlYK2OokIi6JMZ2WwAS43JMmJjj9EKHXxhk0Ssesazmjyqbbvepdv
RMmBKkBbDdFOofbDdAaJjrND9PUupdFfKkLD4KTYFPCJPGgNnBbRLl7TUuIiFJjfMTtMmmtZGgHOoATQqtahzrpjcpfytkdln
MSIiSMBELUuT68OoTLlXxtUHEeLJjJS6MCTtJHhVvCcjYMPOOAaoQqoIWPRrpMVTRBbrYy6Aatvmwipmycmsjlhutlebmssm
IZzTti0FSJj4IS2QqsBb1BSKKLlkkLlVU28OoHIihEYyTSsEeIAZ4GGgLVvlEOoUDdN464TGgtnuegzaiteuvsbisf
8YCcXAaAVKkvLlIiS0ZJjYyDd1UANSWWYyPHhHhpwwGLlJjgFPOZzANEADdAQXWEeSSsLQ0Qqqlswxqaaenaopfsnauzsaxy
OHJICQqcCLKJPEeJjpPpER2TtQXxq154MmrUDduejkSTBbOHYy3ZzLlEFITAaFPWMmJKYAFRrfaykjwpftifehotslcijho
RRDQRrqdrBG31gPpNTMLXBQqVHhBIibvMBWTtIiwQqHJjEeIBbGgD7dKkCcAaMBbmASsaFfiTthbCPpcQqMmTVvtmbxlmtnbr
SHXJjxZzhYyOQUKVEeGNngvkWQqwuMXIIIiJVQYyVQYCL2QqlPAWUHOo0KBbkhuPZZUNnuzF8IJjifzpwapcyqvqvjiixmqos
OPECQqOPpoceQIi9BVZMUDNnKFfLBbHFfhAFWwfBAMW4MLlRrmIKUuFWDdM7XNNSIiJMmDdjsnnxmwfkiwmabalkdumzvbqpo
OoYBbyCBNnbRUNGgHSVv1GgZGLQYF9Yy9LKEeklUuEe3fIZTDdJjtzVYLlTCHhRDOGISIisigMmJjodrctyviyqlgzshnurc
UuEeHhTXBWwMRHXMmSPVQJjDSRSTYSENZR7UurPIiUupRrVvzJHO9AM5OHh3MNRrVJjvnmomaohjnesytsrsdqvpsxhrmbxt
PpYOoFLAFfalfCIicyV2vENDYHhydJAVvSMRINDdOoLlZH2DOoEZLJHhjlPpEeFfze1EHSGUXxIF9fiugshedhznirmsajne
DdXVvHJNnGIJAajYyigj1U2Q7WRrUAa0IZzAaIiKGH6BbA9VPpVvvJREeIYKkPDOoNn4IiPDdDdpdpyirjahgkiuwquhx
PPOUACcaUu8RWWEOFfLlWPVXHPHASIHAahSZzOQqAV6Bb6BTQHESPXVAavPCcAMmapxpsehqtbvaosisahphxvpwoewwruopp
E1eTtAaEPpHhUFfENL3lKkXI3iYYXUuU2KQWwTtTZJHWITCcQOoBbHTCOYyocCFEe3HCchWwfcthqtiwhjztqkuxyyxneue
5OBRVRrvrHL1Ll1FDQQqqYySKkCCETEHVvhKKAaTtkGOogTIiOoWwWQUIFWTBbtPWwXXxDIiVvdxpwfiuqwtketeccsdflhbo
7VHhHJAHA3ahBHFP4B5TtbT5tLLAU4IXAR5TRrtAJjWRr9QqWQqFfHWGPpRBNnOYyXxobrgwhwwaraxiuallpfhbajhv
QqYQKBLS6MCcm5RrPVvPF7XOoxDJjHIDHNnPZKR3CLJNWYLTDQVvqICciCcdtVYWCMmcwyvlywnjlcrkzphdihdfppslbkqy
ORWwrJ3HWUJjuYLYMFF3STF0BTZBEPLlESL24MmlEDdesYHWCRrTOWCcCLSUuWwOoslcwotcwhyepebztbftsffmylywhjo
KkNAanV0VJjQCcKkqWDDF5CNnEXVvBbQqJKGgAQqakZz5DLld3RNnINnRriYTEFfKkDX2O6ZHNnhzoxdetyrjxecfddwvv
RrLPBbJjAYy9FNSsnfa0WXZzNJCJVvjcGFPpKkXBIQAaq5ZXTSM6JjHJMmFDdQqTHQqLGLAalglhtfjhmstxzibxfgjnxwpl
FNnMMFYUKkuy9YyQWIXxHFfhVBDGAXxJMmRA3AaEeX4Xx4xRW7DdSsFEIiefSswEeHNYyCCEec7cnhrarjagdbviwqfmmf
ETtPpFCcUZzUPH0hpAauSEesZzHFGAaMOCPJjp9S7BDFMRBb6KTZ0IiUuJTtjIiZzzPFQYy1Vvqfptkrmfdbscomgfhufe
9YLlTtyFfJBbF3VAYJAQXZHXBBb7OSsEQFDdTt8XPTNEFQTFKKBHPpOXYWUuQqwyxohbkkftqfentpxfqeobxhzxqajyavfj
JKkjNKPJOojIZQKUuSskqzMUumEeiJQIKTSstkOVKOHhBXxEeTNNSKk8Rr3GgLRDd90OAaNYRMmrynorlsnntbokvoiqjpkn
YQqyQUuZTHhtL7LLl6lTZWwDdZFB4BbYSsNZznyZKYykIizCcIiAaAaFB9QJ92CWwARFHSAHOohashfracjqbfbfzztlzq
JjVLlKBbSWCGFfZ5MMmTJFfjSGQ9OoBFZIZzVvizfVvDXYLVPNUuCcENWVMZVvKkzZJD6djzmvwnenpvlyxdbqgstmzgcwskv
HDRrNn6dP0EQqEMmQPNnpJjqIiPFfpOPNnTXxXBLlMOBbom0XxbZzx2MmXXVDQ9FfYyY2LDdlHhXUFfuxyqdvxxtpoeeph
U1JGPUAKkXx6WHYOHPNYyXGEUueYCUuQMMXN5DXxKYykdnYSs1AZ3KMmJWwWZzw7QqjEekzayxmmqcygxnphoyhwaupgju
ANBb8Q6Tt9qQEOoWweqUQYyCcWwLR7rlquYVvKZTU4UPJU2BbMNQTOZzoWLSsGT9t8HCTDdtchglwtqnmujpuutzkyna
CMmJSs5KPpXxMmkTtYyZzVYRryGDdSVABbGNngPQqMYSZXxVIDdiQFfFPNnJFAaTtIifEXxIGZKIikzgiejpfqvzsympavsgvjc
