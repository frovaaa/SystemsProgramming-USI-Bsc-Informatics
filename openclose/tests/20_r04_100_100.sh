#!/bin/bash
exec "$1" "AaBbCcDdEeFfGgJjIiHhKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz" < tests/20_r04_100_100.input
