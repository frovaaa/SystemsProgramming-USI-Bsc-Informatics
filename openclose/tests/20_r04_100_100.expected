ZEXDVN2FS2QTGO4WVV4IORRBK0UQ6qukbrroivvwogtqsfnvdxez
WHQ6XDQXFUGQWKFU0ufkwqgufxqdxqhw
6MJOZHPPTM4ZL5WJ3IXIW38P9KI3iPT0tO49opkpwixijwlzmtpphzojm
LGG7gER2YMMUZ3zumU88ANS5GRXD0dxrgsnaumyregl
KOHW3wh4O0IV1viook
WLBF85HT87th7LTXWUWJYLRTP13BHD06QUDCWGEP25pegwcduqdhbptrlyjwuwxtlfblw
O6Q9qoHP8p3GP0pgh
NEM3Z3z2mFVTOGXOJB6bTBWP2pwbASB4bsatjoxgotvfen
QRVGLLOAXIKG8QOAVWZIUXG3HXC6cxhgxuizwvaoqgkixaollgvrq
2YOZZC60KUQODJITNZNP1COUDSNFC3XXH0hxxcfnsduocpnzntijdoqukczzoy
SQW5wBDXZQ8q1W7wJRWO9owHBLL8llbhrjzxdbqs
SZJJ2YFWFHZEDIUY3yuNRFAW9CVPLBCP6C5cpcblpvcwafrnidezhfwfyjjzs
HIH1WWINCXMS2DEFEHBFST0tsfbhefedsmxcniwwhih
MNTCOTY6TXT56NGUOTPG7INS1snigptougntxtytoctnm
PVI6UV1HTJAGYMKBD3dE3F1TOKVCXX1JT0tjxxcvkotfebkmygajthvuivp
Z4HANVWINT1GOLEYXSTNGPZOXPDIA07aidpxozpgntsxyelogtniwvnahz
IVJST9CMUOEFWQM2MLEADRQDES1FV8vfsedqrdaelmmqwfeoumctsjvi
6PA378PGITSOG7gKH1hkosF8R7CR9VD5dvJSZFY7yfzsjrcrftigpap
KKZGG6gCDXKIMX28LZ3QQ7CUEAHAN8nahaeucqqzlxmikxdcgzkk
ZPRABQ3GSCIITLC53DU4udcltiicsgqbarpz
4CVPMAW1FGCQSSF6WFRL6S8PGD9AYDYWV9vwydyadgpslrfwfssqcgfwampvc
WNFXF1C5cZ6zNCOHE6O7oehocnfxfnw
KFVSYR5VOEVKHHR85WWLPWR8rXWH3hwxwplwwrhhkveovrysvfk
RXGXOT5C4CAEUDBT0tbdueacctoxgxr

UJMFKTPQTE5eBNAW8PDFYZTEGNV2SR3XOD93doxrsvngetzyfdpwanbtqptkfmju
3ZDZEJZYYFMXKC65ckxmfyyzjezdz
WNSJRUNBWQVLBCLEEBZ8G5NZKU1u8kzngzbeelcblvqwbnurjsnw
WWVFILETUUBJFIIFYR7ryfiifjbuutelifvww
R1YYP30VRUY9YCMCMACGTZOPW266wpoztgcamcmcyyurvpyyr
PMQFS26L2NKWAVSO42HDNICIL1licindhosvawknlsfqmp
I4V88PMSPBM7mbp2sLB5blED7dempvi
ZWK0GYL8ZFJYU7uyjfzlygkwz
XQN7IB7b4in8NHRFCXCRDIAZZJUO1oujzzaidrcxcfrhnqx
BNREP8RP9PDDOSYL5NOKMPA8UDWJU8ujwduapmkonlysoddpprpernb
SIPJVA0B0bFD3dfaYD6d72y8vjpis
SEQPD9HD8AMQOAPWYQ16O3oqywpaoqmadhdpqes
4V94QTPESD0d4C1JR0GTM6UJFRBWSJRRBQ9MT9tmqbrrjswbrfjumtgrjcseptqv
PF8HPWSQTKC2OLRC3FHA5ahfcrlocktqswphfp
AKSOKBZJTQK5BA6YFJXKS5QKZY2R4ryWELLKNK1knkllewzkqskxjfyabkqtjzbkoska
7GNW6wnHTI743ithg
11K2RN4Y28ynrk
RJTZOPFSIQ2PGSSSOTEG8QFVQ1qvfqgetosssgpqisfpoztjr
JXFIZKACK6kcYI3ABLB5bHF1fhlbaiyakzifxj
AYMRYOHXXDPPP2OFXMG3JCIZEQC19cqezicjgmxfopppdxxhoyrmya
HCAGJCHLH4AHR4BMTVGR4rgvtmbrhahlhcjgach
UMVZGJSRBOTE7CBXSS8ssxbcetobrsjgzvmu
1QITPFAYDBT9FAFU0GD3dI9USJ5jD6dsuigufaftbdyafptiq
8YJIRYIQBIVMD5EEKTB05btkeedmvibqiyrijy
LMCOPC0CGCO7TKMHIM1DS5sdmihmktocgccpocml
JBRR1AJAB4bRPROMMH8FZVOHMK5kmhovzfhmmorprajarrbj
A9VVHCUL0QLUKTBSKPLEP4peLUHPZF3fzphullpksbtkulqluchvva
1HXTNGCKODFQIPOTBIARQMMMA4ammmqraibtopiqfdokcgntxh
TBRATBMYWAMLTK21kX8AT6V7vtaxtlmawymbtarbt
M311QTI7TYH9hytNUHWPDSOUVAGYZ2zY3yygavuosdpwhunitqm
EZKFIGL2RTMD1WYYWYENH2HFO2ofhhneywyywdmtrlgifkze
ADILZTSAKDECSCMCHVVBLIW4wilbvvhcmcscedkastzlida
WA63DEQSU6P8pJM8TAGRHHLHFMCVNSNO8onsnvcmfhlhhrgatmjusqedaw
UV7PXIWFJY3ZILOUBXGC5IIXYCGKC02S3sckgcyxiicgxbuolizyjfwixpvu
FY5QNHJ0FJDS6NA0FZWNJVCM3TK9GX08xgktmcvjnwzfansdjfjhnqyf
5JLUVJZVF0fvzI8XF0557fxijvulj
TD7dtPOLEIZIL4H59hlizielop
Q0RBEIWM2mHRXF2fxrhwiebrq
0LXPM3C21E0DMP9SAKES5VMB2A48ZC9czabmvsekaspmdecmpxl
QE5LOTGT73tgHZBQQSSFXGNU3Q3qungxfssqqbzhtoleq
ZJ0VMYZJZF3TKD4OITABK3k0batiKV7USS4HQE4eqhssuvkodktfzjzymvjz
VQWCNMNSCLNF2fBUZ6zu9VXKSZN8nzskxvbnlcsnmncwqv
1A3aTQKTWIQKKW7IJN5nj2NB825H7hV4LDA0adlvbniwkkqiwtkqt
RQVNCMAVAV6UCV8XW3wxvcuvavamcnvqr
VD4PK8kEVBB60QYK2k6JMZ2AS439E0esazmjyqbbvepdv
ND9dLD4KTYFPCJPR7rpjcpfytkdln
MSSMBELT68UHLJS6MCYMPIWMVT6tvmwipmycmsjlhutlebmssm
0FS4IS2s1BSVU28ETIAZ4GEUN464nuegzaiteuvsbisf
8YXAS0Z1UANSFPOANEAAQXWSLQ0qlswxqaaenaopfsnauzsaxy
OHJICLKJER2154rejkSTOH3hotslcijho
RBG31gNTMLXBMBHID7dihbmbxlmtnbr
SOQMXIIJVQVQYCL2lPAWUH0huPZF8fzpwapcyqvqvjiixmqos
OPQ9BVZMUDKLABAMW4IKFWM7mwfkiwmabalkdumzvbqpo
CRUNHS1ZGLQYF993fyqlgzshnurc
TXBMRHXSPVQDSRSTYSENZR7rzJHO9AM5O3omaohjnesytsrsdqvpsxhrmbxt
V2vENJASMRINZH2D1EHSGUIF9fiugshedhznirmsajne
XH1U2Q7WU0IKGH6A9JRIYPD4dpyirjahgkiuwquhx
PPOU8RWWEOWPVXHPHASISOAV66vaosisahphxvpwoewwruopp
E1eEUENL3lXI3iYYXU2KQTZJHWITQHTCF3fcthqtiwhjztqkuxyyxneue
5OBHL11lhbo
7VHJAHA3ahBHFP4B5bT5tLLAU4IXAR5AW9waraxiuallpfhbajhv
YQKBLS65PPF7DHIDHPZKR3rkzphdihdfppslbkqy
OJ3HWYLYMFF3STF0BTZBEPESL24lsepebztbftsffmylywhjo
V0VWDDF5CEXJ53RYTEDX2O6oxdetyrjxecfddwvv
LPA9a0WXNJGFXBI5ZXTSM6mstxzibxfgjnxwpl
FMMF9QWIVBDGAJRA3X44xRW7wHNC7cnhrarjagdbviwqfmmf
EFUUPH0hpuHFGMOC9S7BDFMR6KTZ0zPFQ1qfptkrmfdbscomgfhufe
9JF3VAYJAQXZHXB7OEQF8fqeobxhzxqajyavfj
NKPJQIOVKOBTNNS83LR90rlsnntbokvoiqjpkn
QZL7L6lTZZFB4FB9QJ92jqbfbfzztlzq
VKSWCGZ5MTSGQ9BDXYLVPNENWVMZJD6djzmvwnenpvlyxdbqgstmzgcwskv
HD6dP0EEOPTXB0bx2XXVDQ9Y2yqdvxxtpoeeph
U1JGPUA6WHYOHPNXGYCQMMXN5nY1AZ3KJ7jkzayxmmqcygxnphoyhwaupgju
AN8Q69qUQLR7rlquYKZTU4UPJU2MNQTWLGT9t8glwtqnmujpuutzkyna
CJ5jc
